hannah-hjudge/Homework-1/fizzbuzz/app.js

```diff
@@ -0,0 +1,65 @@
+
+// Relying on your newfound knowledge of loops, combine loops and if/else statements together and incrementally build the common fizzbuzz loop.
+
+// - In the loop, every time a number is divisible by **3**, instead of logging the number itself, the word "fizz" should appear.
+// - If the number is divisible by  **5**, the word "buzz" should be logged.
+// - If the number is divisible by both **3** and  **5**, then the word "fizzbuzz" should be logged.
+
+// Follow the steps below.
+
+// ##### Step 1:
+// Construct a for loop that iterates through, and `console.log()`'s out, numbers 1 - 100.
+
+"use strict";
+
+for(let i = 1; i <= 100; i++) {
```

Comment:
Including a space after keywords like `for` is a common practice that can make your code more readable.